lib/ohai/plugins/aix/os.rb

```diff
@@ -25,6 +25,6 @@
 
   collect_data(:aix) do
     os collect_os
-    os_version kernel[:version]
+    os_version shell_out("oslevel -s").stdout.split($/)[0]
```

Comment:
Could do that too; I was just copying the style from the rest of the AIX plugins. You'd prefer `.strip` then I take it?